scripts/installUbuntuDeps.sh

```diff
@@ -3,9 +3,23 @@
 install_apt_deps(){
   ${SUDO} apt-get update -y
   ${SUDO} apt-get install git make gcc g++ libglib2.0-dev docbook2x pkg-config -y
-  ${SUDO} apt-get install libboost-regex-dev libboost-thread-dev libboost-system-dev liblog4cxx-dev rabbitmq-server -y
-  ${SUDO} apt-get install mongodb openjdk-8-jre curl libboost-test-dev nasm yasm gyp libx11-dev libkrb5-dev intel-gpu-tools -y
+  ${SUDO} apt-get install libboost-regex-dev libboost-thread-dev libboost-system-dev liblog4cxx-dev -y
```

Comment:
You updated `installUbuntuDeps.sh`. Then do you need to update `installCentosDeps.sh` too?

Reply:
WebTransport is only tested on Ubuntu, not on CentOS. This workflow runs on Ubuntu only. `installUbuntuDeps.sh` is updated because it failed to install mongo on Ubuntu with apt-get. I'm not sure if yum works. This change only changes `installDeps.sh`'s behavior on GitHub Actions.